lib/galaxy_test/base/api.py

```diff
@@ -83,6 +83,10 @@ def celery_parameters(self):
             "task_default_queue": "galaxy.internal",
         }
 
+    @pytest.fixture(scope="session", autouse=True)
+    def celery_worker_pool(self):
+        return "threads"
```

Comment:
What does this fixture do ?

Reply:
Thanks for catching that. That shouldn't be needed any longer, since the hash is computed manually and not through celery. It may even be the reason for the test failure. Have removed.